### PR TITLE
 style: change margin value for the documentaion button in PolicyEdito…

### LINKED ISSUE #28226: Adding new policy modal has overlapping buttons
Screenshot from 2024-07-27 17-54-12

### PR DESCRIPTION
**Pull request by Mohammed-Yasin-Mulla**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

-  [x] YES

## What kind of change does this PR introduce?

This is a tiny style change to the modal. The documentation button is overlapping with the close button 

## What is the current behavior?

image

## What is the new behavior?

image

#28226

### CODE REVIEW

**Comment by saltcod** [comment]
Thanks @Mohammed-Yasin-Mulla !

Prob makes sense to solve this with padding on the container 
screenshot-2024-07-29-at-21 50 00

**Comment by Mohammed-Yasin-Mulla** [comment]
@saltcod ,
I have reverted the changes and added the `padding` as you mentioned, but I kept the `mt-[-4px]` to make all the elements have the same alignment.

I would love to know if there would be a better solution for `mt-[-4px]` on the button.
image

**Comment by Mohammed-Yasin-Mulla** [comment]
@saltcod ,
I wanted to know if there are any other changes I need to add?
I hope the PR is good to merge.

**Comment by Mohammed-Yasin-Mulla** [comment]
@saltcod , could you please approve the PR if its all good to go

File: apps/studio/components/interfaces/Auth/Policies/PolicyEditorModal/PolicyEditorModalTitle.tsx [hunk 1]
```diff
@@ -61,7 +61,7 @@ const PolicyEditorModalTitle = ({
             Try Supabase Assistant
           </Button>
         )}
-        <Button asChild type="default" icon={<IconExternalLink size={14} />}>
+        <Button asChild type="default" icon={<IconExternalLink size={14} />} className='mt-[-4px]' >
```
Comment (fsansalvadore) [inline apps/studio/components/interfaces/Auth/Policies/PolicyEditorModal/PolicyEditorModalTitle.tsx:64]:
```suggestion
        <Button asChild type="default" icon={<IconExternalLink size={14} />} className='mt-[-4px]'>
```

Comment (fsansalvadore) [reply apps/studio/components/interfaces/Auth/Policies/PolicyEditorModal/PolicyEditorModalTitle.tsx:64]: Prettier is failing, removing that final space should fix it @Mohammed-Yasin-Mulla

**Review by fsansalvadore — DISMISSED** [review]

File: apps/studio/components/interfaces/Auth/Policies/PolicyEditorModal/PolicyEditorModalTitle.tsx (hunk 1 above)
Comment (Mohammed-Yasin-Mulla) [reply apps/studio/components/interfaces/Auth/Policies/PolicyEditorModal/PolicyEditorModalTitle.tsx:64]: thank you

Comment (Mohammed-Yasin-Mulla) [reply apps/studio/components/interfaces/Auth/Policies/PolicyEditorModal/PolicyEditorModalTitle.tsx]: @fabianmossberg, did the refactoring of the code

**Comment by Hallidayo** [comment]
Hi @Mohammed-Yasin-Mulla - Got a conflict on this, once sorted should be all good to go 👍

**Comment by saltcod** [comment]
hey @Mohammed-Yasin-Mulla I accidentally made a mess of your branch, apologies. 

For simplicity, I've created a new brand and cherry-picked your good commit here
https://github.com/supabase/supabase/pull/29229/files

**Comment by Mohammed-Yasin-Mulla** [comment]
Thank you @saltcod  👍🏼

**Pull request closed without merging**